server/src/utils.ts

```diff
@@ -33,7 +33,7 @@ export class Deferred<T> {
 }
 
 export function getTsserverExecutable(): string {
-    return isWindows() ? 'tsserver.cmd' : 'tsserver'
+    return isWindows() ? 'typescript/bin/tsserver.cmd' : 'typescript/bin/tsserver'
```

Comment:
I have only been testing this on Linux, since we are using `require.resolve` the resolver might take care of the windows edge case